packages/core/tests/unit/models/role-test.js

```diff
@@ -0,0 +1,34 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+
+let Store, MetadataService;
+
+module('Unit | Model | role', function(hooks) {
+  setupTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function() {
+    Store = this.owner.lookup('service:store');
+    MetadataService = this.owner.lookup('service:bard-metadata');
+    await MetadataService.loadMetadata();
+  });
+
+  test('Retrieving records', async function(assert) {
+    assert.expect(1);
+
+    const role = await Store.findRecord('role', 'admin');
+    assert.ok(role, 'Found admin role');
+  });
+
+  test('Saving records', async function(assert) {
```

Comment:
We should have tests for updating and deleting roles too

Reply:
I removed the update test because in practice you don't have an attribute to update in the Role model. The `id` which is the name of the role cannot be updated and then createdOn and updatedOn can't also be updated manually.

Reply:
that's fine